src/components/PricingCard/PricingCardButton.react.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import * as React from "react";
+import cn from "classnames";
+
+// TODO: Add color for active button from main PricingCard component
+// TODO: Refactoring
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +RootComponent?: React.ElementType,
+  +active?: boolean,
+|};
+
+function PricingCardButton({
+  className,
+  children,
+  RootComponent,
+  active,
+}: Props): React.Node {
+  const classes = cn("text-center", "mt-6", className);
+  const Component = RootComponent || "a";
+  return active ? (
+    <div className={classes}>
+      <Component className={cn("btn", "btn-green", "btn-block", className)}>
```

Comment:
Rather than calling `cn('btn', 'btn-green'...)` here and on line 31, lets assign the result to a variable above line 23 i.e. `const componentClasses = cn("btn", "btn-green"...);` then use that variable